tensorboard/components/tf_collapsible/tf-collapsible.html

```diff
@@ -0,0 +1,110 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-collapse/iron-collapse.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+
+<!--
+  An element that shows a button that can be used to collapse / expand
```

Comment:
Can you kindly tell me what does not work with https://www.webcomponents.org/element/@polymer/iron-collapse?
If there is a compelling reason, would you mind writing it down as the comment so we can later know what to use when? Thanks